Components/Table/TableHeaderColumn.tsx

```diff
@@ -0,0 +1,40 @@
+import * as React from 'react';
+import styled from 'styled-components';
+import {Column} from './Table';
+
+export interface TableHeaderColumnProps {
+    headerColumnClicked?: {(val: Column): void},
+    column: Column;
+}
+
+export class TableHeaderColumn extends React.Component<TableHeaderColumnProps> {
+
+    render(): JSX.Element {
+
+        const HeaderColumn = styled.th`
```

Comment:
If I'm reading the library correctly, it looks like `styled.whatever` is actually a function that generates a new React component when called.  If that's the case, then we don't want to put it in the `render()` loop.

Instead, it looks like the expected usage is...

```tsx
const HeaderColumn = styled.th`
  cursor: ${props => props.pointerCursor ? "pointer" : "auto"}
  ...
`;

export function TableHeaderColumn(props: TableHeaderColumnProps) {
  return (
    <HeaderColumn pointerCursor={props.headerColumnClicked && props.column.sortProperty} />
  );
}
```